src/template.ts

```diff
@@ -32,11 +32,13 @@ export const DEFAULT_TEMPLATE = `
             *ngFor="let page of p.pages">
             <a tabindex="0" (keyup.enter)="p.setCurrent(page.value)" (click)="p.setCurrent(page.value)" *ngIf="p.getCurrent() !== page.value">
                 <span class="show-for-sr">{{ screenReaderPageLabel }} </span>
-                <span>{{ page.label }}</span>
+                <span *ngIf="page.label && page.label.constructor.name === 'Number'; else notANumber">{{ page.label | number }}</span>
+                <span #notANumber>{{ page.label }}</span>
```

Comment:
This part is not valid Angular template syntax. It should be `<ng-template>`

Reply:
It should be fixed with the latest commit.

Reply:
Yep. looks good, tests passing!